src/main/java/task/Converter.java

```diff
@@ -0,0 +1,18 @@
+package task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Converter {

```

Comment:
I like how you separated the date time conversion logic into a separate `Converter` class. Do you think we could improve it by giving it a more explicit name, to indicate that it's a converter for dates and time?